app/services/save_practice_service.rb

```diff
@@ -24,6 +24,10 @@ def initialize(params)
 
   def save_practice
     begin
+      if @practice_params["support_network_email"].present? && @practice.approved == false
```

Comment:
I am curious as to why this is necessary 🤔

Reply:
So I had practices that were created a while back without a user email... when creating those in Active Admin if there is no user email the practice would not be set to approved.  We've since made a change - so that even if you don't provide a user email in the initial creation the practice gets set to approved.  Since I had old practices that weren't approved initially - whenever I went to publish them.. (even though I had all the required fields set).. I couldn't approve.  TBH that approved flag may be obsolete.. but I thought that there could be practices in the same state on Prod.. i..,e originally created with no user email prior to the change and are not approved - so they would never be able to publish those practices without this code.  Lets talk about this one more... bc like I said the approved flag may be obsolete with the change made to approve practice creation initially.

Reply:
ok I'm going to see how many practices on prod are not approved

Reply:
there is only 1 practice on prod that is not approved and it isn't published currently... so I think this code can be removed and if there are any issues like this we can do a one-time data fixup. What are your thoughts on this, brad? 🤔

Reply:
Thats fine I can remove that check - If they try to publish that practice - we may just have to go in and manually approve it.   Going forward no practices should fall in to this state.